### ISSUE TITLE
Lilac server won't open / is blank, operations can't be applied.

### ISSUE THREAD

**Issue by TheodoreGalanos** [lilacai/lilac#640]
Hello!

The latest fix allowed me to install and create a project (lilac = 0.17)

However, after creating an initial test object, loading a JSON file (correctly, if I'm reading the .yaml file right) I can't seem to load the server on the browser. Console seems to show `Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain"` but not sure that's relevant.

Additionally, running operations on it gives errors like 

```
pydantic.error_wrappers.ValidationError: 1 validation error for EmbeddingConfig
embedding
  Signal "gte_small" not found in the registry (type=value_error)
```

My apologies if I missed something obvious, I tried to look around issues.

**Comment by nsthorat** [comment]
Hi @TheodoreGalanos, been trying to reproduce this with no luck! I'm on it today.

Can you share your OS version, python version, and potentially the JSON dataset you're trying to load (if it's not private)?

And did you `pip install lilac` or `pip install lilac[all]`?

**Comment by TheodoreGalanos** [comment]
Hello @nsthorat !

Apologies for late reply. I am on Windows 10 and in a python 3.11 environment. I installed lilac with `pip install[gte]`. I will give `all` a try now.

**Comment by nsthorat** [comment]
Ah awesome, thanks. I'll boot up my windows machine today and look into this!

**Comment by nsthorat** [comment]
@TheodoreGalanos one last question -- are you running this in WSL or from the windows terminal?

**Comment by TheodoreGalanos** [comment]
Hey, I'm running this from a miniconda environment.

**Comment by TheodoreGalanos** [comment]
BTW it worked fine through WSL. Wonder if it's a windows thing?

**Comment by nsthorat** [comment]
Yeah it must be a windows bug.. got my windows machine set up but it's got some issues. Still going to work on this!

Does WSL unblock you or do you need it to work without that?
